src/BloomBrowserUI/bookEdit/toolbox/readers/readerToolsModel.ts

```diff
@@ -1055,10 +1055,16 @@ export class ReaderToolsModel {
                 .closest("body")
                 .children('.qtip:not(".uibloomSourceTextsBubble")')
                 .remove();
-
+            // If the Format (Styles) dialog is showing, then we don't want to create
+            // bookmarks.  The div#format-toolbar is instantiated only when the dialog
+            // is showing.  See BL-13043.
+            const formatDialog = $(editableElements[0])
+                .closest("body")
+                .find("#format-toolbar");
```

Comment:
I'm slightly concerned about this fix because the user can actually type in the book while the style dialog is open.

What are the ramifications of not setting these bookmarks? If it is simply that the cursor jumps, we can probably just live with that.
I just didn't know if it might cause data problems (like introduction of unwanted characters, etc.).